### PR TITLE
Delete icon theme

### PR DESCRIPTION
**Pull request by dhonx**
icon theme will replace the user theme. which is not good, so it will be better to remove it

### CODE REVIEW

**Comment by alexesprit** [comment]
> icon theme will replace the user theme. which is not good, so it will be better to remove it

It's isn't replaced actually, but after installation VS Code asks to change icon theme.

On the other side, the built-in theme is pretty useless: it does not apply V icon, but "overwrites" the whole theme, and contains only one icon.

I think it should be removed. An alternative way to add V icon for respective files is to contribute to icon theme projects.

**Comment by Alhadis** [comment]
> An alternative way to add V icon for respective files is to contribute to icon theme projects.

I help maintain [`file-icons/vscode`](https://github.com/file-icons/vscode), which already has a V icon, so :+1: from me.

File: package.json [hunk 1]
```diff
@@ -150,5 +143,10 @@
 		"vscode:prepublish": "npm run compile",
 		"compile": "tsc -p ./",
 		"watch": "tsc -watch -p ./"
+	},
+	"__metadata": {
+		"id": "336b4b95-caeb-446b-8072-ffe7908230a9",
+		"publisherDisplayName": "0x9ef",
+		"publisherId": "9afb706b-1f4e-4984-98cb-a48d44cddc9f"
```
Comment (Alhadis) [inline package.json]: I don't think you meant to add this. 😉

Comment (dhonx) [reply package.json]: hahaha, yeah I forgot that  😁

**Pull request merged**